card/mod/04_settings/lib/card/setting.rb

```diff
@@ -1,29 +1,34 @@
 # -*- encoding : utf-8 -*-
 
-
 class Card
-
-  # Used to extend setting modules like Card::Set::Self::Create in the settings mod
+  # Used to extend setting modules like Card::Set::Self::Create in the
+  # settings mod
   module Setting
     # Let M = Card::Setting           (module)
     #     E = Card::Set::Self::Create (module extended with M)
-    #     O = Card["*create"]         (object)
-    attr_accessor :codename                                                # accessible in E
-    mattr_accessor :groups, :group_names, :user_specific                   # accessible in E and M
-    def self.extended(host_class)
-       host_class.mattr_accessor :restricted_to_type, :rule_type_editable  # accessible in E and O
+    #     O = Card['*create']         (object)
+    # accessible in E
+    attr_accessor :codename
+    # accessible in E and M
+    mattr_accessor :groups, :group_names, :user_specific
+    def self.extended host_class
+      # accessible in E and O
+      host_class.mattr_accessor :restricted_to_type, :rule_type_editable
     end
 
     @@group_names = {
-      templating:  "Templating",
-      permission:  "Permissions",
-      webpage:     "Webpage",
-      pointer:     "Pointer",
-      editing_cue: "Editing cues",
-      event:       "Events",
-      other:       "Other"
+      templating:  'Templating',
+      permission:  'Permissions',
+      webpage:     'Webpage',
+      pointer:     'Pointer',
+      editing_cue: 'Editing cues',
+      event:       'Events',
+      other:       'Other',
+      config:      'Config'
     }
-    @@groups = @@group_names.keys.each_with_object({}) { |key, groups| groups[key] = [] }
+    @@groups = @@group_names.keys.each_with_object({}) do |key, groups|
+                 groups[key] = []
+               end
```

Comment:
`end` at 31, 15 is not aligned with `@@groups = @@group_names.keys.each_with_object({}) do |key, groups|` at 29, 4.